app/app/urls.py

```diff
@@ -175,6 +175,8 @@
     url(r'^extension/chrome?', retail.views.browser_extension_chrome, name='browser_extension_chrome'),
     url(r'^extension/firefox?', retail.views.browser_extension_firefox, name='browser_extension_firefox'),
     url(r'^extension/?', retail.views.browser_extension_chrome, name='browser_extension'),
+    path(r'funder', retail.views.how_it_works_funder, name='how_it_works_funder'),
```

Comment:
This will fail.  If you're using `r''`, you'll want to switch this to `re_path()` - Otherwise, simply drop the `r` here.

Reply:
Why this will fail?

Reply:
@darkdarkdragon The docs [example](https://docs.djangoproject.com/en/2.0/topics/http/urls/#example) should shed some light on it

Reply:
@thelostone-mc sorry, still don't understand. can you please point me to exact problem?

Reply:
@mbeacom can you please explain what is the problem here?